templates/repo/issue/view_content/comments.tmpl

```diff
@@ -601,6 +601,22 @@
 				</span>
 			</div>
 			{{end}}
+		{{else if eq .Type 31}}
+			{{if not $.UnitProjectsGlobalDisabled}}
+			<div class="timeline-item event" id="{{.HashTag}}">
+				<span class="badge">{{svg "octicon-project"}}</span>
+				{{template "shared/user/avatarlink" dict "user" .Poster}}
+				<span class="text grey muted-links">
+					{{template "shared/user/authorlink" .Poster}}
+					{{$newProjectDisplayHtml := .CommentMetaData.ProjectTitle|Safe}}
```

Comment:
XSS.

In most case, do not use "Escape" or "Safe" in templates.

Reply:
I think I just copied them from Type 30.

Reply:
Type 30 never does `{{$newProjectDisplayHtml := .CommentMetaData.ProjectTitle|Safe}}`

Reply:
1dfb66c706eac2d54c49203b49deb95f3f5265cc